src/components/icons/IconSend.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+function IconSend (props) {
+  
+  const defaultProps = {
+		width: 24,
+		height: 24
+	};
+
+  props = {...defaultProps, ...props};
+ 
+  return (
+        <svg {...props} width="24" height="24" viewBox="0 0 24 24" fill="#0657F9" xmlns="http://www.w3.org/2000/svg">
+                <g  clip-path="url(#clip0_1_1348)">
```

Comment:
clip-path  should be renamed clipPath
fill-rule  should be renamed fillRule
clip-rule  should be renamed clipRule

These currently raise error in the console when the code is run. This happens because of the switch from regular HTML where properties like `clip-path` are correct to React Components / JSX  where `clipPath` etc are correct.